checkstyle.xml

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0"?>
```

Comment:
@AnakinSklavenwalker Do you maybe want to add the changes we discussed in the meeting?

Reply:
iirc i already added them